README.md

```diff
@@ -1,2 +1,93 @@
 # pylops-mpi
-MPI-powered PyLops with MPI4Py  Topics Resources - GSOC2023 Project
+[![pages-build-deployment](https://github.com/PyLops/pylops-mpi/actions/workflows/pages/pages-build-deployment/badge.svg)](https://github.com/PyLops/pylops-mpi/actions/workflows/pages/pages-build-deployment)
+[![PyLops-MPI](https://github.com/PyLops/pylops-mpi/actions/workflows/build.yml/badge.svg)](https://github.com/PyLops/pylops-mpi/actions/workflows/build.yml)
+![OS-support](https://img.shields.io/badge/OS-linux,win,osx-850A8B.svg)
```

Comment:
Small comment, unlikely this will work on windows... maybe for now we should just have linux as I suspect this is where you develop (and where tests are run)?

Reply:
As the tests are working on both mac and ubuntu, it does work on both.
For windows I think it should work but will have to check.
Changed to `linux, osx` as of now...